<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>774</width>
    <height>361</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cam project</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="search_button">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QPushButton" name="connect_button">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QPushButton" name="disconnect_button">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Disconnect</string>
    </property>
   </widget>
   <widget class="QComboBox" name="combo_port">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>40</y>
      <width>86</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="combo_baud">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>40</y>
      <width>86</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>170</y>
      <width>361</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QSlider" name="roll">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>10</y>
      <width>31</width>
      <height>81</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-90</number>
    </property>
    <property name="maximum">
     <number>90</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="Get_angle_button">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>130</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Get Angle</string>
    </property>
   </widget>
   <widget class="QSlider" name="pitch">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>10</y>
      <width>31</width>
      <height>81</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-90</number>
    </property>
    <property name="maximum">
     <number>90</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSlider" name="yaw">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>10</y>
      <width>31</width>
      <height>81</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-90</number>
    </property>
    <property name="maximum">
     <number>90</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
